watertap/examples/flowsheets/mvc/mvc_single_stage_ui.py

```diff
@@ -0,0 +1,649 @@
+        ui_units=pyunits.Pa,
+        display_units="Pa",
+        rounding=2,
+        description="Distillate pump pressure change",
+        is_input=True,
+        input_category="Distillate pump",
+        is_output=False,
+    )
+
+    # System costing
+    exports.add(
+        obj=fs.costing.utilization_factor,
+        name="Utilization factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TIC,
+        name="Practical investment factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Practical investment factor - [total investment cost/direct "
+        "capital costs]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.electricity_cost,
+        name="Electricity cost",
+        ui_units=fs.costing.base_currency / pyunits.kWh,
+        display_units="$/kWh",
+        rounding=3,
+        description="Electricity cost",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.factor_total_investment,
+        name="Total investment factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.heat_exchanger.material_factor_cost,
+        name="Heat exchanger material cost factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Heat exchanger material cost factor",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.evaporator.material_factor_cost,
+        name="Evaporator material cost factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Evaporator material cost factor",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.compressor.unit_cost,
+        name="Compressor unit cost",
+        ui_units=pyunits.USD_2001,
+        display_units="$(2001)",
+        rounding=2,
+        description="Compressor unit cost",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+
+    # Outlets
+    feed_salinity = 1e3 * fs.feed.properties[0].mass_frac_phase_comp["Liq", "TDS"]
```

Comment:
Not sure why we're manually converting units here.

Reply:
I think the issue is that we're going from dimensionless (`mass_frac_phase_comp`) to g/kg, so I think this manual conversion is necessary? This is also how it's done in the `def display_metrics` in the MVC flowsheet itself

Reply:
Seems okay on my system...

<img width="642" alt="Screenshot 2023-12-14 at 1 37 38 PM" src="https://github.com/watertap-org/watertap/assets/30801372/19280a60-cb17-49f2-bf6e-acb93efe6c45">

Reply:
Alright, I've procrastinated long enough. I automatically convert the units and change `flowsheet=None` to just `flowsheet`